<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_novavenda</class>
 <widget class="QDialog" name="frm_novavenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vendas</string>
  </property>
  <widget class="QTableWidget" name="tw_listaprodutos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>741</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_total">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>617</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Venda:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_totalvenda">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>617</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R$</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>300</y>
     <width>61</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Qtde</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_qtde">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>300</y>
     <width>144</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Código do produto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_codigopeca">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>651</y>
     <width>347</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_excluirproduto">
      <property name="text">
       <string>Excluir Produto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_finalizarvenda">
      <property name="text">
       <string>Finalizar Venda</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tw_listapecas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>97</y>
     <width>741</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>357</y>
     <width>761</width>
     <height>253</height>
    </rect>
   </property>
   <property name="title">
    <string>Itens venda:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>761</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Seleção de peças:</string>
   </property>
   <widget class="QWidget" name="layoutWidget_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Filtrar por:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_vd_filtrar"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>58</y>
      <width>391</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="txt_vd_filtrar"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_pesquisarproduto">
       <property name="text">
        <string>Pesquisar produto</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Qtde. Estoque</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_qtdeEstoque">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_adicionarItem">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>325</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Lucro:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_totallucro">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>650</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R$</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>tw_listapecas</zorder>
  <zorder>tw_listaprodutos</zorder>
  <zorder>lb_total</zorder>
  <zorder>lb_totalvenda</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>btn_adicionarItem</zorder>
  <zorder>label_5</zorder>
  <zorder>lb_totallucro</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
